### PR TITLE
Create health habit use cases

### PR DESCRIPTION
**Pull request by aloomdar**

### CODE REVIEW

File: src/use-cases/health-habit.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const result = db
+        .update(healthHabits)
+        .set({
+            date: input.date,
+            waterIntake: input.waterIntake,
+            sleepHours: input.sleepHours,
+            mealsEaten: input.mealsEaten
+        })
+        .where(
+            eq(healthHabits.patientId, input.patientId)
```
Comment (jesuskcapellan) [inline src/use-cases/health-habit.ts]: Shouldn't we be updating a specific health habit entry? This will update every entry from that patient.

File: src/use-cases/health-habit.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        throw new Error("Could not updated")
+    }
+
+}
+
+export async function createHealthHabit(input:CreateHealthHabitsInput): Promise<CreateHealthHabitsResult> {
+    const result = await db
+        .insert(healthHabits)
+        .values({
+            date: sql`${input.date}`,
```
Comment (jesuskcapellan) [inline src/use-cases/health-habit.ts]: Why did you need to do sql? I'm assuming there's an error but I don't know about it :/

**Review by jesuskcapellan — CHANGES_REQUESTED** [review]

File: src/use-cases/health-habit.ts (hunk 2 above)
Comment (aloomdar) [reply src/use-cases/health-habit.ts]: I did it to fix one bug at first then tried to fix another and forgot to remove this

File: src/use-cases/health-habit.ts (hunk 1 above)
Comment (aloomdar) [reply src/use-cases/health-habit.ts]: Added habitId

**Review by jesuskcapellan — APPROVED** [review]

**Review by tanvirlongia — APPROVED** [review]
